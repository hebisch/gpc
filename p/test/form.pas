program test(output);

        procedure foo0(procedure bar);
                begin
                bar
                end;

        procedure foo1(x :integer; procedure bar(i :integer));
                begin
                bar(x)
                end;

        procedure zero;
                begin
                writeln('I take no parameters');
                end;

        procedure one(z :integer);
                begin
                writeln('My argument is ',z:1);
                end;

        begin
        foo0(zero);
        foo1(12, one);
        writeln('success');
        end.
