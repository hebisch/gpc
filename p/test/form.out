I take no parameters
My argument is 12
success
